src/lib.rs

```diff
@@ -505,15 +503,16 @@ impl<T: Runtime> Client<T> {
 
     /// Subscribe to new blocks.
     pub async fn subscribe_blocks(&self) -> Result<Subscription<T::Header>, Error> {
-        let headers = self.rpc.subscribe_blocks().await?;
+        let headers: Subscription<T::Header> = self.rpc.subscribe_blocks().await?;
```

Comment:
don't understand why this was needed, should be possible to deduce ^^

Reply:
Compiles on my machine :sunglasses: